data_server.py

```diff
@@ -0,0 +1,62 @@
+#Module to load dataset and divide it into train, test and validation sets for X and Y
+
+def Load_dataset(filename):
+
+    import numpy as np
+
+    try:
+        file = open(filename)
+    except:
+        print("Unable to open the file")
+    else:
+        print("File opened")
+
+    X_list = []
+    Y_list = []
+
+    first_line = True
+
+    print("Preparing dataset...")
+
+    for line in file:
+        if first_line == True:          #Skipping first line of file which contains name of data columns
+            first_line = False
+        else:
+            row = line.split(',')
+            Y_list.append(int(row[0]))
+            # Pixel values for each picture are saved in csv in single cells as a string containing values splited by
+            # space-bar so we need to split that string to substrings, convert them to ints and add these pixel values to list one by one
+            X_list.append([int(pixel) for pixel in row[1].split()])
+
+    X_data = np.array(X_list)
+    Y_data = np.array(Y_list)
+
+    file.close()
```

Comment:
As above - use with(...) and you can skip it. It is more common practice not to have to remember about closing.